src/main/java/com/github/elenterius/biomancy/block/MeatsoupCauldronBlock.java

```diff
@@ -198,13 +217,13 @@ else if (level < MAX_LEVEL - 3 && item.isIn(ModTags.Items.RAW_MEATS)) {
 				return ActionResultType.func_233537_a_(worldIn.isRemote);
 			}
 			else if (level >= MAX_LEVEL - 3 && level < MAX_LEVEL - 1) {
-				if (item instanceof PotionItem) {
+				if (item instanceof PotionItem || item == ModItems.REJUVENATING_MUCUS.get()) {
 					Potion potion = PotionUtils.getPotionFromItem(stack);
-					if (potion == Potions.HEALING || potion == Potions.REGENERATION) {
+					if (potion == Potions.HEALING || potion == Potions.REGENERATION|| item == ModItems.REJUVENATING_MUCUS.get()) {
 						if (!worldIn.isRemote) {
 							if (!player.abilities.isCreativeMode) {
 								stack.grow(-1);
-								if (stack.isEmpty()) {
+								if (stack.isEmpty() || item != ModItems.REJUVENATING_MUCUS.get()) {
```

Comment:
has to be &&
imagine a mucus stack that is empty returning bottles

Reply:
uhhhh

Reply:
fixed